compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -471,7 +471,7 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
     ungated!(no_builtins, CrateLevel, template!(Word), WarnFollowing, EncodeCrossCrate::Yes),
     ungated!(
         target_feature, Normal, template!(List: r#"enable = "name""#),
-        DuplicatesOk, EncodeCrossCrate::No,
+        DuplicatesOk, EncodeCrossCrate::Yes,
```

Comment:
I think this was accidentally left enabled

Reply:
Indeed, removed.